src/events/message/index.ts

```diff
@@ -24,45 +27,82 @@ export default class MessageEvent extends Event {
 		}
 
 		if (message.attachments.size > 0) {
-			const file = message.attachments.first(); // only take first attachment as, normally, users cannot send more than one attachment
+			const file =
+				message.attachments.find((attachment) => {
+					if (!attachment.name || attachment.width) {
+						return false;
+					}
+
+					const fileExtension = extname(attachment.name).substring(1);
+					const language = fileExtension || "txt";
+
+					return language === "txt" || extensions.has(language);
+				}) || message.attachments.first(); // only take first attachment as, normally, users cannot send more than one attachment
 
-			if (!file?.name || file?.width) {
+			if (!file?.name) {
 				return;
 			}
 
 			const fileExtension = extname(file.name).substring(1);
 			const language = fileExtension || "txt";
-			if (language !== "txt" && !extensions.has(language)) {
+
+			if (language !== "txt" && !extensions.has(language) && !message.content.trim()) {
 				return;
 			}
 
-			const code = await fetch(file.url)
-				.then((res) => res.text())
-				.catch(() => undefined);
+			// prettier-ignore
+			// There are conflicts between Prettier and ESLint
+			const code =
+				language === "txt" || extensions.has(language)
+					? await fetch(file.url)
+						.then((res) => res.text())
+						.catch(noop)
+					: undefined;
+
+			const processed =
+				message.content.split("\n", MAX_LINES).length === MAX_LINES
+					? await processContent(message.content)
+					: undefined;
+			const content = code?.trim() ? await createBin(code, language).catch((e: Error) => e) : undefined;
 
-			if (!code?.trim()) {
+			if (!content && !processed) {
 				return;
 			}
 
-			const content = await createBin(code, language).catch((e: Error) => e.message);
-			const processed = await processContent(message.content);
+			if (content instanceof Error) {
+				// eslint-disable-next-line max-len
+				const botMessage = `${content}\n\nCependant, bien que votre message n'ait pas été effacé, il a été jugé trop "lourd" pour être lu (code trop long, fichier texte présent). Nous vous conseillons l'usage d'un service de bin pour les gros morceaux de code, tel ${process.env.BIN_URL!.slice(
+					0,
+					-4,
+				)}`;
+
+				await message.channel.send(botMessage).catch(noop);
 
-			if (processed) {
-				sendBinEmbed(message, processed, (embed) => embed.addField("📁 Attachement", content));
-			} else {
-				sendBinEmbed(message, content);
+				return;
 			}
+
+			const otherAttachments = message.attachments.filter((attachment) => attachment.id !== file.id);
+
+			await sendBinEmbed(
+				message,
+				processed || message.content.trim(),
+				content ? (embed): MessageEmbed => embed.addField("📁 Pièce jointe", content) : undefined,
+				otherAttachments.size ? otherAttachments : undefined,
```

Comment:
```suggestion
				otherAttachments.size > 0 ? otherAttachments : undefined,
```
c'est mieux quand c'est explicite imo

Reply:
soit